k8s/manifest.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: otel-collector
+  labels:
+    app: opentelemetry
+    component: otel-collector
+spec:
+  ports:
+  - name: otlp-grpc # Default endpoint for OpenTelemetry gRPC receiver.
+    port: 4317
+    protocol: TCP
+    targetPort: 4317
+  selector:
+    component: otel-collector
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: otel-collector
+  labels:
+    app: opentelemetry
+    component: otel-collector
+spec:
+  serviceAccountName: otel-collector
+  selector:
+    matchLabels:
+      app: opentelemetry
+      component: otel-collector
+  minReadySeconds: 5
+  progressDeadlineSeconds: 120
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: opentelemetry
+        component: otel-collector
+    spec:
+      containers:
+      - command:
+          - "/otelcol-custom"
+          - "--config=/conf/otel-config.yaml"
+        image: %OTEL_COLLECTOR_IMAGE%
```

Comment:
When the image is replaced, it doesn't seem to take the registry into account.  My image is replaced with `otelcol-custom:latest`, and I get this error:

`Warning  Failed             21s (x3 over 59s)  kubelet             Failed to pull image "otelcol-custom:latest": rpc error: code = Unknown desc = failed to pull and unpack image "docker.io/library/otelcol-custom:latest": failed to resolve reference "docker.io/library/otelcol-custom:latest": pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed`